src/panels/config/storage/dialog-mount-view.ts

```diff
@@ -225,12 +276,29 @@ class ViewMountDialog extends LitElement {
       `ui.panel.config.storage.network_mounts.errors.${error}`
     ) || error;
 
+  private _computeWarningCallback = (warning: string): string =>
+    this.hass.localize(
+      // @ts-ignore
+      `ui.panel.config.storage.network_mounts.warnings.${warning}`
```

Comment:
Maybe something like this?

```ts
      this._existing ? "The selected CIFS version is not recommended to use" : "We could not connect to your network share, maybe you have to change the version to make it work?"
```

Reply:
That is 2 different things.
"We could not connect" is an error that happens after you try connecting.
The warning is for when you select an insecure version.

Reply:
With that said, I want to change the connection error message in a new PR after this to add a button to get to the host logs.
I can also extend that message to include notes about trying a different version.

Reply:
> That is 2 different things. "We could not connect" is an error that happens after you try connecting. The warning is for when you select an insecure version.

Agree it should be in the catch:
```ts
      this._validationWarning.cifs_version = "could_not_connect";
```